zcash_client_backend/src/constants/regtest.rs

```diff
@@ -0,0 +1,38 @@
+/// The testnet coin type for ZEC, as defined by [SLIP 44].
+///
+/// [SLIP 44]: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+pub const COIN_TYPE: u32 = 1;
+
+/// The HRP for a Bech32-encoded testnet [`ExtendedSpendingKey`].
+///
+/// Defined in [ZIP 32].
```

Comment:
This is not defined in ZIP 32; it is defined in the `zcashd` codebase ([here](https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L496)), because regtest mode is a `zcashd`-specific mode.

Reply:
Done